plugins/registry/combine.go

```diff
@@ -0,0 +1,55 @@
+package registry
+
+import (
+	"github.com/laszlocph/woodpecker/model"
+)
+
+type combined struct {
+	registries []model.ReadOnlyRegistryService
+	dbRegistry model.RegistryService
+}
+
+func Combined(dbRegistry model.RegistryService, registries ...model.ReadOnlyRegistryService) model.RegistryService {
+	registries = append(registries, dbRegistry)
+	return &combined{
+		registries: registries,
+		dbRegistry: dbRegistry,
+	}
+}
+
+func (c combined) RegistryFind(repo *model.Repo, name string) (*model.Registry, error) {
+	for _, registry := range c.registries {
+		res, err := registry.RegistryFind(repo, name)
+		if err != nil {
+			return nil, err
+		}
+		if res != nil {
+			return res, nil
+		}
+	}
+	return nil, nil
+}
+
+func (c combined) RegistryList(repo *model.Repo) ([]*model.Registry, error) {
+	var registeries []*model.Registry
+	for _, registory := range c.registries {
```

Comment:
Shouldn't the call be propagated to `dbRegistries` too?

Reply:
Shouldn't have to, dbRegistry is added to the list https://github.com/laszlocph/woodpecker/pull/120/files#diff-60ca29804ed147e90820993d6f198f1fR13